### PR TITLE
feat: Make the prelude available in description page

### PR DESCRIPTION
**Pull request by YoanwM**
This PR aims to add a Prelude pane in the `/description` view of an exercise (if its `prelude.ml` is nonempty).

Just to recall, this is intended to be used with [learn-ocaml-mode](https://github.com/pfitaxel/learn-ocaml.el), where the browser will only display questions (no code editor).

### CODE REVIEW

File: static/css/learnocaml_description.css [hunk 1]
```diff
@@ -52,14 +53,97 @@ body {
   overflow: auto;
   flex: 1 3 auto ;
 }
-#learnocaml-exo-tab-meta > h1::after {
+#learnocaml-exo-tabQ > * > h1::after {
```
Comment (erikmd) [inline static/css/learnocaml_description.css:56]: Hi @YoanwM, thanks!
I just read your PR code (not tested yet) and it looks quite good.
Just one question for the moment: is this change expected?
(`#learnocaml-exo-tab-meta` → `#learnocaml-exo-tabQ`)

File: src/app/learnocaml_common.ml [hunk 2]
```diff
@@ -905,6 +905,8 @@ module Editor_button (E : Editor_info) = struct
       sync_exercise token id ~editor:(Ace.get_contents E.ace) >|= fun _save -> ()
 end
 
+
+
```
Comment (erikmd) [inline src/app/learnocaml_common.ml:909]: nitpick: these extra newlines are unneeded I guess ;)

File: static/css/learnocaml_description.css (hunk 1 above)
Comment (YoanwM) [reply static/css/learnocaml_description.css:56]: I can't remember I did this, I think it's just a mistake I didn't see !

File: src/app/learnocaml_common.ml (hunk 2 above)
Comment (YoanwM) [reply src/app/learnocaml_common.ml:909]: Yes I just move a fonction in an other part of the documents and forgot to erase these lines

**Comment by yurug** [comment]
It would be nice to have a description for this PR.

**Comment by erikmd** [comment]
Rebasing this PR…

**Review by erikmd — APPROVED** [review]
@yurug 

> It would be nice to have a description for this PR.

Done.

I think this PR can now be merged.

FWIW, below are some screenshots in four different situations.

## with a prelude / full mode

2021-09-21_Screenshot_description_prelude_full

## with a prelude / responsive mode

2021-09-21_Screenshot_description_prelude_responsive

## with no prelude / full mode

2021-09-21_Screenshot_description_no-prelude_full

## with no prelude / responsive mode

2021-09-21_Screenshot_description_no-prelude_responsive

**Pull request merged**